lib/ansible/modules/service.py

```diff
@@ -80,7 +80,7 @@
     use:
         description:
         - The service module actually uses system specific modules, normally through auto detection, this setting can force a specific module.
-        - Normally it uses the value of the 'ansible_service_mgr' fact and falls back to the old 'service' module when none matching is found.
+        - Normally it uses the value of the C(ansible_service_mgr) fact and falls back to the old I(service) module when none matching is found.
```

Comment:
```suggestion
        - Normally it uses the value of the C(ansible_service_mgr) fact and falls back to the old M(ansible.legacy.service) or M(ansible.builtin.service) module, in that order, when none matching is found.
```

Reply:
@s-hertel like this?

Reply:
I don't know, I assumed M() wouldn't work for `ansible.legacy`. Does that link to anything?

Since `ansible.legacy.service` is a superset of `ansible.builtin.service`, it's redundant to list both. It would be better to link other docs on it if necessary (like https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#what-is-the-difference-between-ansible-legacy-and-ansible-builtin-collections) than go into depth on for it for an individual plugin.

Reply:
I don't know whether it links to anything either, but I agree that this is going way deeper than needed for one module documentation - moreover, if we were to add a link like that for this module, arguably we should do that for all modules in the `ansible.builtin``  "collection", which is clearly overkill.
I think a simple reference to `ansible.builtin.service` should suffice.
```suggestion
        - Normally it uses the value of the C(ansible_service_mgr) fact and falls back to the old M(ansible.builtin.service) module when none matching is found.
```
Implicitly, anything in the `ansible.builtin.*` space can be overridden with something in `ansible.legacy.*`.

Reply:
That's backwards; `ansible.legacy.*` means that it can be overridden, `ansible.builtin.*` explicitly means that it cannot.

Reply:
Referring to `ansible.builtin.service` would change the meaning and no longer reflect what the action plugin does. It loads the legacy module, which may resolve to the builtin.

What about
```suggestion
        - Normally it uses the value of the C(ansible_service_mgr) fact and falls back to the C(ansible.legacy.service) module when none matching is found.
```

@samccann @oraNod do you have any suggestions on the formatting for legacy modules?

Reply:
FWIW I feel like `C()` is the right choice and would be consistent with RST formatting for modules, legacy or otherwise. To be clear about what I'm basing this on, I'm going from [the docs](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#format-macros-within-module-documentation) that state `C()` is for monospace (code) text.